content/articles/run-length-encoding/index.md

```diff
@@ -0,0 +1,150 @@
+### Introduction
+
+In this article, we will look at how Run Length Encoding works in helping us reduce our file sizes.
+Before delving into the topic, we will first get to know what is compression.
+
```

Comment:
It is always good to add `Table of contents` section and a `Prerequisities` section.